modalsheet/src/main/java/eu/wewox/modalsheet/ModalSheet.kt

```diff
@@ -68,15 +69,61 @@ fun ModalSheet(
     },
     content: @Composable ModalSheetScope.() -> Unit,
 ) {
+    ModalSheet(
+        data = Unit,
+        visible = { visible },
+        onDismiss = onDismiss,
```

Comment:
Implementation-wise I think that "data sheet" should depend on the regular "boolean visible sheet". Not the other way around